addons/mail/models/mail_tracking_value.py

```diff
@@ -120,11 +116,21 @@ def _create_tracking_values(self, initial_value, new_value, col_name, col_info,
 
     def _tracking_value_format(self):
         """ Return structure and formatted data structure to be used by chatter
-        to display tracking values.
+        to display tracking values. Order it according to asked display, aka
+        ascending sequence (and field name).
 
         :return list: for each tracking value in self, their formatted display
           values given as a dict;
         """
+        if not self:
+            return []
+        field_models = self.field.mapped('model')
+        if len(set(field_models)) > 1:
```

Comment:
otherwise `field_models[0]` can technically crash below
```suggestion
        if len(set(field_models)) != 1: